CONTRIBUTING.rst

```diff
@@ -3,118 +3,111 @@ Contributing to skrub
 
 First off, thanks for taking the time to contribute!
 
-The following is a set of guidelines for contributing to
-`skrub <https://github.com/skrub-data/skrub>`__.
+Below are some guidelines to help you get started.
 
-|
-.. contents::
-   :local:
 
-|
+Have a question?
+----------------
 
-I just have a question
-----------------------
+If you have any questions, feel free to reach out:
 
-We use GitHub Discussions for general chat and Q&As. `Check it
-out! <https://github.com/skrub-data/skrub/discussions>`__
+- Join our community on `Discord <https://discord.gg/ABaPnm7fDC>`_ for general chat and Q&A.
+- Alternatively, you can `start a discussion on GitHub <https://github.com/skrub-data/skrub/discussions>`_.
 
-What should I know before I get started?
-----------------------------------------
+What to know before you begin
+-----------------------------
 
-To understand in more depth the incentives behind skrub,
-read our `vision statement. <https://skrub-data.org/stable/vision.html>`__
-Also, if scientific literature doesn't scare you, we greatly
-encourage you to read the two following papers:
+To understand the purpose and goals behind skrub, please read our
+`vision statement. <https://skrub-data.org/stable/vision.html>`_
 
-- `Similarity encoding for learning
-  with dirty categorical variables <https://hal.inria.fr/hal-01806175>`__
-- `Encoding high-cardinality string categorical
-  variables <https://hal.inria.fr/hal-02171256v4>`__.
+If you're interested in the research behind skrub,
```

Comment:
I'm not sure I'd mention this in the "What to know before you begin" section, maybe in a "to go further" somewhere else ?

Reply:
It used to be there, so I didn't challenge that idea. While I think highlighting the research background of the project is helpful, I also think the GapEncoder and MinHashEncoder are slightly less central to skrub than they used to be. This needs to be discussed with the team.